include/engine/entity.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file entity.h
+ * @author Grégory Brivady
+ * @brief Master object of the game
+ * @version 0.1
+ * @date 2022-12-03
+ * 
+ * 
+ */
+#ifndef ENGINE_ENTITY_H
+#define ENGINE_ENTITY_H
+
+#include <string>
+#include <vector>
+
+#include <SFML/Graphics/Drawable.hpp>
+#include <SFML/Graphics/CircleShape.hpp>
+
+#include "engine/physic/physic.h"
+#include "engine/physic/phxbox.h"
+#include "engine/box/box.h"
+
+class InputHandler;
+
+/**
+ * @brief An entity is an object that exists in the game universe.
+ * 
+ * @details This class both links to all the properties of an object, 
+ * but also handles interfacing between the properties. Each property is stored
+ * as a pointer. To add a property to an entity, one should use the add functions,
+ * and once everything is initialised, the entity can be added to the universe.
```

Comment:
it's "initialized", isn't it? :thinking:

Reply:
As I said in another comment, I believe it is correct in British English - and it was I used in most of the project; Does not really matter anyway

Reply:
then i prefer the american english :wink: 

and no it does not matter at all, let's leave it that way :yum: